_data/rules.yml

```diff
@@ -350,7 +350,7 @@ groups:
                   This rule can be very noisy in dynamic infra with legitimate container start/stop/deployment.
               - name: Container High CPU utilization
                 description: Container CPU utilization is above 80%
-                query: '(sum(rate(container_cpu_usage_seconds_total{name!=""}[3m])) BY (instance, name) * 100) > 80'
+                query: '(sum(rate(container_cpu_usage_seconds_total{pod!=""}[3m])) by (instance, pod) / sum(container_spec_cpu_quota / container_spec_cpu_period) by (instance, pod) * 100) > 80'
```

Comment:
FYI:
i am running container with docker and the metric from cadvisor does not have the label "pod" on my instance.

Reply:
Are you running containers in k8s ?

In my setup, out of k8s, I see "name" instead of "pod".

Reply:
(I am running in k8s; not knowing what @Kariton is running on)

Reply:
Looks like i need to clarify. 😄 

I run it on docker - not k8s.

Reply:
so i also have "name" instead of "pod".